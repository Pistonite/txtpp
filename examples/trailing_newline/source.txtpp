If --no-trailing-newline is passed, no trailing newline will be added in the output.

This flag has no effect on include and temp.

TXTPP#include include

-TXTPP#temp temp.out
- temp files should not be affected
+TXTPP#temp temp.out.trailing
+ temp files should not be affected. this will have trailing because there is an empty line after it
+
end of file