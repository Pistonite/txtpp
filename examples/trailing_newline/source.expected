If --no-trailing-newline is passed, no trailing newline will be added in the output.

This flag has no effect on include and temp.

this include file has no trailing new line
end of file